targets/rs485_usage_test/work/f2/C/src/f2.c

```diff
@@ -0,0 +1,30 @@
+/* Body file for function f2
+ * Generated by TASTE (kazoo/templates/skeletons/c-body/function.tmplt)
+ * You can edit this file, it will not be overwritten
+
+    !! IMPORTANT                                                        !!
+    !! When you modify your design (interface view), you must update    !!
+    !! the procedures corresponding to the provided interfaces in this  !!
+    !! file. The up-to-date signatures can be found in the header file. !!
+*/
+#include "f2.h"
+
+static asn1SccMyInteger myStaticInt = 1;
+
+void f2_startup(void)
+{
+}
+
+void f2_PI_query
+      (const asn1SccMyInteger *IN_p1)
+
+{
+  asn1SccMyInteger arg = *IN_p1;
+  arg++;
+  f2_RI_response(&arg);
+}
+
+void f2_PI_trigger( void )
```

Comment:
If this interface is unused, it can be safely removed.

Reply:
done